### ISSUE TITLE
Issue running chown/permission_error on cygwin locally

### ISSUE THREAD

**Issue by lydia-duncan** [chapel-lang/chapel#12756]
Note: I've seen the relevant JIRA issue and this is different.

On Cygwin, if I run test/library/standard/Filesystem/fsouza/chown, I get 
```
[Executing diff permission_error.good permission_error.exec.out.tmp]
1c1
< uncaught PermissionError: SYS_ERR (in chown with path "file.txt")
---
> uncaught SystemError: SYS_ERR (in chown with path "file.txt")
[Error matching program output for library/standard/FileSystem/fsouza/chown/permission_error]
```

I suspect that when we updated for error handling, we missed that Cygwin behaved differently than Linux for this error due to its different behavior when run via nightly testing versus locally (see JIRA 81).

Because of that JIRA issue, I'm not sure it is worth updating the test to have a cygwin specific .good file, but I figured we should track it in case someone else runs into the issue.

**Comment by bradcray** [comment]
I don't think we should have an open issue that we don't intend to do anything about, so think we should either close this as a "won't fix" or (preferably) add a cygwin-specific .good file if this is the expected behavior or a .future and/or .suppressif that explains the current behavior if it's not.

**Comment by lydia-duncan** [comment]
Sorry, should have been a bit more specific.  I don't know that it is worth updating the test right now, but it seems wrong that it is in this current state and I don't know how else to track that and really don't have the mental cycles to remind myself exactly how cygwin specific .good files work.
